src/services/storage/Storage.ts

```diff
@@ -0,0 +1,83 @@
+import { isDefined } from '@prefecthq/prefect-design'
+import { reactive } from 'vue'
+import { StorageItem } from '@/services/storage/StorageItem'
+
+type Store<T extends StorageItem> = Map<string, T | undefined>
```

Comment:
Totally expected this to be the return of pinia. Guess its still dead lol

Reply:
lol I was just thinking last week about how long it had been since I'd built a store based app. This is 100% ui business logic. The `reactive` map is the only thing that makes this vue 😄